### PR TITLE
PyImportSortBear.py: Adds asciinema

### PR DESCRIPTION
**Pull request by SerekKiri**
This pull request adds asciinema to PyImportSortBear.

Closes https://github.com/coala/coala-bears/issues/876

### CODE REVIEW

File: bears/python/PyImportSortBear.py [hunk 1]
```diff
@@ -17,6 +17,7 @@ class PyImportSortBear(LocalBear):
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
     CAN_FIX = {'Formatting'}
+    ASCIINEMA_URL = 'https://asciinema.org/a/0vqWoZBXyh8IpkMLhBnF5YWmB'
```
Comment (li-boxuan) [inline bears/python/PyImportSortBear.py]: Please make it public 👍

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala-bears/pull/2787/commits/25bd79a54eb0e90c03085e091fd1757bd38b3239

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**